sources/install/package_web.sh

```diff
@@ -922,7 +922,22 @@ function package_web() {
     install_soapui                  # SoapUI is an open-source web service testing application for SOAP and REST
     install_sqlmap                  # SQL injection scanner
     install_sslscan                 # SSL/TLS scanner
+    install_Postman                 # Postman - API platform for testing APIs
     end_time=$(date +%s)
     local elapsed_time=$((end_time - start_time))
     colorecho "Package web completed in $elapsed_time seconds."
 }
+
+function install_Postman() {
+    colorecho "Installing postman"
```

Comment:
can you add : # CODE-CHECK-WHITELIST=add-aliases

Reply:
Done